<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Button_open">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wybierz plik</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_okna">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Kaisera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hamminga</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Barletta</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_nfft">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>WÄ…skopasmowy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Szerokopasmowy</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_st_nachodzenia">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_generate">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wygeneruj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_save">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Zapisz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_spektogram">
       <property name="text">
        <string>Spektogram</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="k" stdset="0">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
